files/MAWhoNeedToRenew.php

```diff
@@ -46,6 +46,10 @@
   $simplifiedRegistrationEvents = keepOnlyActualRegistrations($simplifiedRegistrationEvents);
 }
 
+if(isset($_GET["json"])) {
```

Comment:
I don't understand those lines: why would we get such field?

Reply:
Because of https://github.com/vhouchouas/membership-scripts/blob/0d7189791bb6f884cf702f73f1a47149518ce68a/files/js.js#L147.
I reuse `files/MAWhoNeedToRenew.php` to get the helloasso registrations (JSON formatted).

Reply:
ah ok, I see. Thanks for the explanation!